tdee_macro_calc/main.py

```diff
@@ -1,12 +1,19 @@
+from curses import panel
 import json
+from datetime import datetime, date, timedelta
+from operator import add
+from xmlrpc.client import Boolean
```

Comment:
Use the `Boolean` type from Python's standard library `typing` for type annotations.

```
from typing import Boolean
```

Reply:
Good catch. I noticed a few imports that weren't necessary but forgot to go back and clean them up. I'm not sure how that one in particular snuck in there, VS Code must've thrown it in when I wasn't looking somehow.